multidict/_multidict_base.py

```diff
@@ -95,10 +83,6 @@ def _viewbaseset_sub(view, other):
 def _viewbaseset_xor(view, other):
     if not isinstance(other, Iterable):
         return NotImplemented
-    if isinstance(view, Set):
```

Comment:
I think only the first condition is duplicate ?

Reply:
It's more that it checks if it's a set and then converts it to a set. But, this seems to be required to work properly...